frontend/packages/console-shared/src/utils/helm-utils.ts

```diff
@@ -0,0 +1,22 @@
+import * as _ from 'lodash';
+import { Model, NodeModel } from '@patternfly/react-topology';
+import { K8sResourceKind, K8sResourceKindReference } from '@console/internal/module/k8s';
+
+interface OdcNodeModel extends NodeModel {
+  resource?: K8sResourceKind;
+  resourceKind?: K8sResourceKindReference;
+}
+
+export const isHelmResource = (resource: K8sResourceKind): boolean =>
+  resource?.metadata?.labels?.['app.kubernetes.io/managed-by'] === 'Helm' ||
+  resource?.metadata?.labels?.['heritage'] === 'Helm' ||
+  _.has(resource?.metadata?.labels, 'helm.sh/chart');
+
+export const isHelmResourceInModel = (resource: K8sResourceKind, model: Model): boolean => {
+  if (!isHelmResource(resource)) {
+    return false;
+  }
+  return !!model.nodes.find((node) => {
+    return (node as OdcNodeModel).resource?.metadata?.uid === resource?.metadata?.uid;
+  });
+};
```

Comment:
This is topology specific and shouldn't be in console-shared.
I don't see where this function is even referenced other than from dev-console.